benchmark/benchmark.py

```diff
@@ -15,6 +16,14 @@
 SKIP = set(['fft', 'hyantes', 'README'])
 
 
+def print_entry(name, res):
+    print(" - ", name)
+    print(' '*4, end='')
+    for name, dt in res.items():
```

Comment:
maybe `sorted(res.items())` to make sure it's always in the same order between runs?

Reply:
Since in py3.7 dicts are ordered by default, this prints them in insertion order which is "native", "firefox", "chrome".